src/scope.rs

```diff
@@ -20,6 +21,11 @@ use std::path::PathBuf;
 #[derive(Trace, Finalize)]
 pub enum Scope {
     Root(PathBuf),
+    Normal {
```

Comment:
Normal seems to be an odd term here to me.

Reply:
Agreed. I've now renamed it to `Let`. Does that sound better?